CODE_OWNERSHIP.md

```diff
@@ -0,0 +1,55 @@
+# Code Ownership and AMP
+
+This document describes the code ownership model for the AMP open source
+project, and in particular the [AMP HTML GitHub project](https://github.com/ampproject/amphtml).
+
+The goals of enforcing AMP code ownership are as follows:
+
+* Provide the flexibility to specify a different set of owners for individual
+  directories / files in the repository
+* Allow for the recursive inheritance of ownership for files / directories based
+  on the ownership of parent directory(ies)
+* Prevent incoming pull requests from being merged until they are approved by a
+  sufficient number of owners so as to cover all files / directories in the
+  change
+* Allow overall project / area owners to approve the odd PR, without their
+  having to receive a GitHub notification them about every incoming PR
+* Provide an automated mechanism to update the approved status of a PR each time
+  a reviewer modifies it
+* Forbid non-committers and non-owners from modifying the approved status of a
+  pull request
+* Make it easy to add / change the list of owners for a given directory / file
+
+## Chromium's code ownership model
+
+The Chromium open source project implements a code ownership
+[model](https://chromium.googlesource.com/chromium/src/+/master/docs/code_reviews.md#OWNERS-files)
+that we would like to use for AMP HTML. Unfortunately, with GitHub's
```

Comment:
I don't understand this sentence right here. Maybe pull out a section titled

"Why we don't use GitHub's code ownership solution"

Reply:
Done.